book/src/external-provisioner.md

```diff
@@ -8,7 +8,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-provisioner v2.0.4](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v2.0.4) | [release-2.0](https://github.com/kubernetes-csi/external-provisioner/tree/release-2.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-provisioner:v2.0.4 | v1.17 | - | v1.19
+[external-provisioner v2.1.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v2.1.0) | [release-2.1](https://github.com/kubernetes-csi/external-provisioner/tree/release-2.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-provisioner:v2.1.0 | v1.17 | - | v1.19
 [external-provisioner v1.6.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.6.0) | [release-1.6](https://github.com/kubernetes-csi/external-provisioner/tree/release-1.6) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | quay.io/k8scsi/csi-provisioner:v1.6.0 | v1.13 | - | v1.18
```

Comment:
This one should be updated to https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.6.1